src/js/engagement_view/src/tests/engagementView/data/graphVizData.tsx

```diff
@@ -1,85 +1,76 @@
 export const vizGraphData = [
 	{
-		uid: 40,
-		node_key: "53e2abc7-ce21-4be0-b70a-4c835eefecd1",
+		uid: 10005,
+		node_key: "3db5c6b8-1c4e-414f-9947-7a02f07156fd",
+		dgraph_type: ["Process"],
+		process_name: "cmd.exe",
+		process_id: 5824,
+		children: [
+			{
+				uid: 10032,
+				node_key: "69c90caa-cbc6-4b80-8f29-60f501aeb114",
+				dgraph_type: ["Process"],
+				process_name: "svchost.exe",
+				process_id: 6132,
+			},
+		],
+		risks: [
+			{
+				uid: 10133,
+				dgraph_type: ["Risk"],
+				node_key: "Suspicious svchost",
+				analyzer_name: "Suspicious svchost",
+				risk_score: 75,
+			},
+			{
+				uid: 10138,
+				dgraph_type: ["Risk"],
+				node_key: "Rare Parent of cmd.exe",
+				analyzer_name: "Rare Parent of cmd.exe",
+				risk_score: 10,
+			},
+		],
+	},
+	{
+		uid: 10013,
+		node_key: "DESKTOP-FVSHABR",
```

Comment:
Changes like these seem irrelevant to this PR title and description. Should this be a in a separate PR?

Reply:
Fixed (2), yeah idk about (1) lemme look into this.